docs/guides/running-a-poc.mdx

```diff
@@ -0,0 +1,223 @@
+    
+    A typical role breakdown for a Statsig implementation looks something like:
+    
+    **Traditional Team Structure**
+    
+    - **Engineer/Data - Role:** Implementation and Orchestration
+        - SDK implementation on server/web/mobile
+        - Data pipelining metrics into Statsig
+            - SDK, warehouse ingestion, data tool integrations
+        - Feature/Experiment orchestration within the code base
+        - Data analysis on the console UI
+    - **Product - Role:** Planning and Execution
+        - Plan and Implement features/experiments in Statsig console
+        - Validate test results
+    
+- What parts of our tech stack will host the solution and can Statsig support those services?
+    
+    Statsig is a full-stack solution so we can cover your client, server, and mobile applications in most cases. Ensure the systems you are implementing internally are supported by Statsig. We also [integrate](https://docs.statsig.com/integrations/introduction) with a lot of internal tools that customers are already using today!
+    
+- Define the **timeline** for the POC
+    
+    In order to ensure that you are keeping yourself and your team accountable, we recommend setting a timeline of 2-4 weeks for your proof of concept. This allows you to quickly get up to speed testing and understanding the software, but also ensures you scope the work properly to measuring the value and impact of Statsig in a timely manner.
+    
+    **Example Timeline:**
+    
+    **Weeks 1-2**
+    
+    - Instrument Statsig SDK into your app
+    - Set up metric ingestion pipelines
+    - Begin rollouts
+
+ 
+    **Weeks 3-4**
+    
+    - Finalize rollouts
+    - Collect Data
+    - Analyze Results
+    - Finalize POC
+
+- How will we **get help** when we need it?
+    
+    We pride ourselves in providing [excellent customer support](https://statsig.com/blog/why-people-love-statsig-customer-support). At Statsig there is no formal ticketing process for support, instead the [slack community](https://statsig.com/community) gives you access to the engineers & data scientists who build the product. This format not only allows for fast response times, but also increases the likelihood of productive discourse and community engagement. We get some of our best feedback there!
+    
+    Since you’re already here, [our documentation](https://docs.statsig.com) makes getting information on implementation easy so feel free to start exploring! The search function is also particularly useful.
+    
+    Find some suggested starting points for each role below:
+    
+    **Engineer/Data**
+    
+    - [Client SDK Docs](https://docs.statsig.com/client/introduction)
+    - [Server SDK Docs](https://docs.statsig.com/server/introduction)
+    - [Metrics Overview](https://docs.statsig.com/metrics)
+
+    **Product**
```

Comment:
I'm an MDX Noob but this is indenting improperly, does anyone know how to fix this gracefully? 

![image](https://github.com/statsig-io/docs/assets/125311112/40c1916a-d3d2-436a-8ab8-33670d2e6c59)